docsrc/admin/git.md

```diff
@@ -0,0 +1,52 @@
+# Configuring Git
```

Comment:
There are also git configuration instructions in the install guide. That's for configuring deploy keys for use by the update script, this is for configuring personal keys for use by the administrator. Might need to explain that.